ui/vic-ui-h5c/vic/src/vic-app/src/app/services/vm-view.service.ts

```diff
@@ -83,7 +80,8 @@ export class VicVmViewService {
                 try {
                     for (let objectId in data.data) {
                         if (data.data.hasOwnProperty(objectId)) {
-                            vchsArray.push(new VirtualContainerHost(data.data[objectId]));
+                            vchsArray.push(
+                                new VirtualContainerHost(data.data[objectId]));
```

Comment:
is it necessary to create a new class instance here each time? you could make an interface and just make sure the data conforms to the interface type.